src/core/net/dns_types.hpp

```diff
@@ -133,16 +133,16 @@ class Header : public Clearable<Header>
     }
 
     /**
-     * Defines types of query.
+     * Defines types of query (OpCode).
      */
     enum QueryType : uint8_t
     {
-        kQueryTypeStandard = 0,
-        kQueryTypeInverse  = 1,
-        kQueryTypeStatus   = 2,
-        kQueryTypeNotify   = 4,
-        kQueryTypeUpdate   = 5,
-        kQueryTypeDso      = 6,
+        kQueryTypeStandard = 0,  ///< Query
+        kQueryTypeInverse  = 1,  ///< IQuery
+        kQueryTypeStatus   = 2,  ///< Status
+        kQueryTypeNotify   = 4,  ///< Notify
+        kQueryTypeUpdate   = 5,  ///< Update
+        kQueryTypeDso      = 6,  ///< DNS Stateful Operations
```

Comment:
To fix pretty check:
```suggestion
        kQueryTypeStandard = 0, ///< Query
        kQueryTypeInverse  = 1, ///< IQuery
        kQueryTypeStatus   = 2, ///< Status
        kQueryTypeNotify   = 4, ///< Notify
        kQueryTypeUpdate   = 5, ///< Update
        kQueryTypeDso      = 6, ///< DNS Stateful Operations
```

Reply:
Ok, done in the latest push.